### PR TITLE
Introduce ephemeral-storage quotas

### PR DESCRIPTION
**Pull request by bastjan**
See https://kb.vshn.ch/appuio-cloud/references/quality-requirements/performance/resource-quota.html
(27.9.2021)
Or: https://github.com/appuio/appuio-io-docs/blob/2e11cb7729bc157e70778ced4e1ff52515d0504e/docs/modules/ROOT/pages/references/quality-requirements/performance/resource-quota.adoc

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -65,12 +71,19 @@ parameters:
           max:
             cpu: "1"
             memory: "4G"
+            # limits.ephemeral-storage
+            ephemeral-storage: "500Mi"
           min:
             cpu: "10m"
             memory: "4Mi"
+            ephemeral-storage: "100Ki"
```
Comment (ccremer) [inline class/defaults.yml:79]:
```suggestion
            ephemeral-storage: "5Mi"
```

**Review by ccremer — CHANGES_REQUESTED** [review]
Please link the KB article mentioned

**Comment by bastjan** [comment]
> Please link the KB article mentioned, to the PR description

https://github.com/appuio/component-appuio-cloud/pull/19/files#diff-6fd07ea8eb879a940d3dfb5e2d4f7bcb6f049795bc05fbd7f2dbbb7bc2f5c310L41

It's already mentioned a bit higher up.

File: class/defaults.yml (hunk 1 above)
Comment (bastjan) [reply class/defaults.yml:79]: Why set the minimum to such a high value?

Comment (ccremer) [reply class/defaults.yml:79]: Well, is it that high? 
Mainly matching the value in `defaultRequest`

Question back: Why only 100Ki?

**Comment by ccremer** [comment]
> to the PR description

> It's already mentioned a bit higher up.

Yeah, I meant it would be nice to have a short summary in the PR. So if someone looks at the changelog and clicks on this PR, one would not really need to comb through (collapsed) code when figuring out what has changed and why.

**Comment by bastjan** [comment]
> > to the PR description

Sorry i missread 🤦‍♀️

File: class/defaults.yml (hunk 1 above)
Comment (bastjan) [reply class/defaults.yml:79]: > Question back: Why only 100Ki?

It's off the cuff. A pod not writing to its containers file systems and has no empty dir storage should use almost no ephemeral storage.

The other min values are quite low too...

Comment (ccremer) [reply class/defaults.yml:79]: I don't quite understand what you mean, but I have no hard feeling either, so :shrug:

**Review by ccremer — APPROVED** [review]

**Review by glrf — APPROVED** [review]

**Pull request merged**